pkg/controller/v1beta1/inferenceservice/reconcilers/keda/keda_reconciler.go

```diff
@@ -105,6 +106,17 @@ func getKedaMetrics(componentExt *v1beta1.ComponentExtensionSpec,
 						trigger.Metadata["namespace"] = metric.External.Metric.Namespace
 					}
 				}
+				if triggerType == "opentelemetry" {
+					trigger.Type = "external"
+					trigger.Metadata["clampMin"] = strconv.Itoa(int(minReplicas))
+					trigger.Metadata["clampMax"] = strconv.Itoa(int(maxReplicas))
+					trigger.Metadata["metricQuery"] = query
+					trigger.Metadata["targetValue"] = strconv.Itoa(targetValue)
```

Comment:
I guess we have got almost all the keys for the metadata, except there is another one called `operationOverTime`. The value for this key will define, over certain time period, what operation we can leverage to aggregate the metrics.

For example, `operationOverTime: "avg"` means during the past 60 seconds for instance, we pick up the average value for all the same metrics.

`operationOverTime: "last_one"` means during the past 60 seconds for instance, we pick up the last metric record no matter how many we received.

There are other operation options for this key as well.
Suggest you can add it.